metricbeat/Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN \
 RUN pip3 install --upgrade pip==20.1.1
 RUN pip3 install --upgrade setuptools==47.3.2
 RUN pip3 install --upgrade docker-compose==1.23.2
+RUN go get gotest.tools/gotestsum
```

Comment:
I think we need to control the version of this tool, if we do something like adding the installation as dependency of the mage targets we wouldn't need to add it here in docker images and other build scripts.